content/en/docs/examples/virtual-machines/single-network/index.md

```diff
@@ -1,59 +1,71 @@
-typically requires a VPC or a VPN, as well as a container network that
-provides direct (without NAT or firewall deny) routing to the endpoints. The machine
-is not required to have access to the cluster IP addresses assigned by Kubernetes.
+- Virtual machines (VMs) must have IP connectivity to the endpoints in the mesh.
+  This typically requires a VPC or a VPN, as well as a container network that
+  provides direct (without NAT or firewall deny) routing to the endpoints. The
+  machine is not required to have access to the cluster IP addresses assigned by
+  Kubernetes.
 
-* Mesh expansion VMs must have access to a DNS server that resolves names to cluster IP addresses. Options
-include exposing the Kubernetes DNS server through an internal load balancer, using a Core DNS
-server, or configuring the IPs in any other DNS server accessible from the VM.
+- VMs must have access to a DNS server that resolves names to cluster IP
+  addresses. Options include exposing the Kubernetes DNS server through an
+  internal load balancer, using a Core DNS server, or configuring the IPs in any
+  other DNS server accessible from the VM.
 
-* Install the [Helm client](https://docs.helm.sh/using_helm/). Helm is needed to enable mesh expansion.
+- Install the [Helm client](https://docs.helm.sh/using_helm/). Helm is needed to
+  support adding VMs to your mesh.
 
 The following instructions:
 
-* Assume the expansion VM is running on GCE.
-* Use Google platform-specific commands for some steps.
+- Assume the expansion VM is running on GCE.
+- Use Google platform-specific commands for some steps.
 
 ## Installation steps
 
 Setup consists of preparing the mesh for expansion and installing and configuring each VM.
 
-### Preparing the Kubernetes cluster for expansion
+### Preparing the Kubernetes cluster for VMs
 
-The first step when adding non-Kubernetes services to an Istio mesh is to configure the Istio installation itself, and
-generate the configuration files that let mesh expansion VMs connect to the mesh. To prepare the
-cluster for mesh expansion, run the following commands on a machine with cluster admin privileges:
+The first step when adding non-Kubernetes services to an Istio mesh is to
+configure the Istio installation itself, and generate the configuration files
+that let VMs connect to the mesh. To prepare the cluster for mesh
+expansion, run the following commands on a machine with cluster admin
+privileges:
 
-1.  Ensure that mesh expansion is enabled for the cluster. If you didn't use
-    the `--set global.meshExpansion.enabled=true` flag when installing Helm,
-    you can use one of the following two options depending on how you originally installed
-    Istio on the cluster:
+1.  Ensure that the `mesh expansion` option is enabled for the cluster. If you
+    didn't use the `--set global.meshExpansion.enabled=true` flag when
+    installing Helm, you can use one of the following two options depending on
```

Comment:
We've deprected the Helm docs, this needs to be recast in terms of istioctl.

Reply:
Yes, but that will be part of the next iteration. This iteration is just  replacing mesh expansion with VM-based alternatives.